libsrc/hyperion/Hyperion.cpp

```diff
@@ -190,8 +191,6 @@ void Hyperion::handleSettingsUpdate(const settings::type& type, const QJsonDocum
 	}
 	else if(type == settings::LEDS)
 	{
-		QMutexLocker lock(&_changes);
```

Comment:
All the functions protected by this mutex (handleSettingsUpdate and update) is called from Hyperion's own thread. No need to synchronize.